microprofile/tracing/src/main/java/module-info.java

```diff
@@ -25,7 +25,8 @@
 @Feature(value = "Tracing",
         description = "MicroProfile tracing spec implementation",
         in = HelidonFlavor.MP,
-        path = "Tracing"
+        path = "Tracing",
+        since = "4.0.0"
```

Comment:
The correct annotation structure should be:
```java
@Feature(value = "Tracing",
        description = "MicroProfile tracing spec implementation",
        in = HelidonFlavor.MP,
        path = "Tracing",
        since = "1.0.0")
@Deprecated(since="4.0.0", forRemoval=true)
```

Marking this feature to be valid since version 1.0.0 to 4.0.0 and now deprecated.
If our feature processing creates wrong results, we will need to fix it - this is the intended behavior (sorry I did not explain it right when you asked about this)

Reply:
Fixed. Fixed `@Deprecated` `since` File handler.